package.json

```diff
@@ -27,8 +27,11 @@
   },
   "homepage": "https://github.com/JonasImm/fullstack-template#readme",
   "dependencies": {
+    "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "json-server": "^0.16.1",
+    "jsonwebtoken": "^8.5.1",
```

Comment:
I do not find any usage of `jsonwebtoken` -> it's not used and should not be in this pull request.

Reply:
Good point. Will fix it